### PR TITLE
Update notebooks to 0.4.0

### PR DESCRIPTION
**Pull request by jbe456**
Upgrading the 3 notebooks to 0.4.0

### CODE REVIEW

File: retail/pricing-simulations-around-product-classes/main.ipynb [hunk 1]
```diff
@@ -29,20 +29,14 @@
...
      "output_type": "stream",
      "text": [
-      "Welcome to Atoti 0.3.1!\n",
-      "\n",
-      "By using this community edition, you agree with the license available at https://www.atoti.io/eula.\n",
-      "Browse the official documentation at https://docs.atoti.io.\n",
-      "Join the community at https://www.atoti.io/register.\n",
-      "\n",
-      "You can hide this message by setting the ATOTI_HIDE_EULA_MESSAGE environment variable to True.\n"
+      "Deleting existing \"Unnamed\" session to create the new one.\n"
```
Comment (SamTaz) [inline retail/pricing-simulations-around-product-classes/main.ipynb:39]: I don't really understand this message, I don't have it when I re-run the notebook.

File: retail/pricing-simulations-around-product-classes/main.ipynb [hunk 2]
```diff
@@ -661,20 +663,31 @@
...
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 21,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'http://localhost:37399/#/dashboard/326'"
```
Comment (SamTaz) [inline retail/pricing-simulations-around-product-classes/main.ipynb:683]: Really weird, first time I opened this dashboard all the margin values were negative. All the later times worked fine, I cannot reproduce anymore.

File: retail/pricing-simulations-around-product-classes/main.ipynb (hunk 1 above)
Comment (jbe456) [reply retail/pricing-simulations-around-product-classes/main.ipynb:39]: yes it's because I run the cell twice, I'll update it

**Comment by jbe456** [comment]
Regarding the undefined caption, it's an existing bug in 0.3.1: https://github.com/atoti/atoti/issues/30

**Pull request merged**